joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -253,6 +375,38 @@ JointTrajectoryController::on_configure(const rclcpp_lifecycle::State & previous
 
   last_state_publish_time_ = lifecycle_node_->now();
 
+  // action server configuration
+  {
+    if (lifecycle_node_->has_parameter("allow_partial_joints_goal")) {
```

Comment:
if you declare a parameter with a default value (`false` in this case) I think you can directly ask for the parameter value and don't have to ask if the parameter exists.

Reply:
fixed aec9f2d

Reply:
fixed aec9f2d